Week01/Problem05/shashemi_05.py

```diff
@@ -0,0 +1,19 @@
+def LCM(start,end):
```

Comment:
it's often a good idea to put a docstring in your functions to describe a bit about what they do and how they work see link below:
https://www.geeksforgeeks.org/python-docstrings/